docs/source/geopm_prof.3.rst

```diff
@@ -147,6 +143,17 @@ initialized.
   progress.  This method signals the completion of one work unit out
   of the total passed to ``geopm_tprof_init()``.
 
+OMPT Integration
+----------------
+
+Integrating with OMPT provides automation for region identification, entry,
```

Comment:
Should we add details or a link on how to integrate with ompt?

Reply:
It's mostly automatic.  If a code uses OMP, and OMPT support is enabled in the Runtime (today, it is by default), then OMPT regions will appear in the report.  There's nothing else for the user to do unless they want progress reporting, which you discuss below.